code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -44,14 +44,15 @@
 // Reuse the cache/code from stools, todo maybe unify.
 /obj/structure/bed/on_update_icon()
 	..()
```

Comment:
Note to self: parent call does `overlays.Cut()`, will not cooperate with these changes.